engine/src/main/java/org/hibernate/validator/internal/xml/stax/mapping/ContainerElementTypeStaxBuilder.java

```diff
@@ -36,12 +36,11 @@
 import org.hibernate.validator.internal.util.logging.LoggerFactory;
 import org.hibernate.validator.internal.xml.ClassLoadingHelper;
 import org.hibernate.validator.internal.xml.ContainerElementTypePath;
-import org.hibernate.validator.internal.xml.binding.ContainerElementTypeType;
 import org.hibernate.validator.internal.xml.stax.AbstractStaxBuilder;
 import org.hibernate.validator.internal.xml.stax.mapping.ContainerElementTypeConfigurationStaxBuilder.ContainerElementTypeConfiguration;
 
 /**
- * Builds the cascading and type argument constraints configuration from the {@link ContainerElementTypeType} elements.
+ * Builds the cascading and type argument constraints configuration from the {@code <container-element-type>} elements.
```

Comment:
Minor nitpick but just above in this commit, you replaced it with a link to `ContainerElementTypeStaxBuilder`, let's be consistent.